src/transformers/models/depth_anything/configuration_depth_anything.py

```diff
@@ -139,6 +146,8 @@ def __init__(
         self.fusion_hidden_size = fusion_hidden_size
         self.head_in_index = head_in_index
         self.head_hidden_size = head_hidden_size
+        self.depth_estimation_type = depth_estimation_type
+        self.max_depth = max_depth if max_depth else 1
```

Comment:
Could we use a smarter default here? Given 20 is for indoor models, that might make more sense.

Reply:
It actually has to be 1 by default, if we are not using a _metric_ depth model, because in that case, the output shouldn't be scaled by `max_depth`. So, if `depth_estimation_type` is `relative`, it should be 1

Reply:
Ah, yes, you're right!